tests/components/motionmount/test_entity.py

```diff
@@ -0,0 +1,87 @@
+"""Tests for the MotionMount Entity base."""
+
+from unittest.mock import MagicMock
+
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import device_registry as dr, entity_registry as er
+from homeassistant.helpers.device_registry import format_mac
+
+from . import MAC, ZEROCONF_NAME
+
+from tests.common import MockConfigEntry
+
+
+async def test_entity(
+    hass: HomeAssistant,
+    mock_config_entry: MockConfigEntry,
+    mock_motionmount_config_flow: MagicMock,
+) -> None:
+    """Tests the state attributes."""
+    mock_config_entry.add_to_hass(hass)
+
+    mock_motionmount_config_flow.name = ZEROCONF_NAME
+    mock_motionmount_config_flow.mac = MAC
+    mock_motionmount_config_flow.is_authenticated = True
+    mock_motionmount_config_flow.error_status = 0
+    assert await hass.config_entries.async_setup(mock_config_entry.entry_id)
+
+    assert hass.states.get("sensor.my_motionmount_error_status").state == "none"
+
+
+async def test_entity_no_mac(
+    hass: HomeAssistant,
+    mock_config_entry: MockConfigEntry,
+    mock_motionmount_config_flow: MagicMock,
+) -> None:
+    """Tests the state attributes."""
+    mock_config_entry.add_to_hass(hass)
+
+    mock_motionmount_config_flow.name = ZEROCONF_NAME
+    mock_motionmount_config_flow.mac = b"\x00\x00\x00\x00\x00\x00"
+    mock_motionmount_config_flow.is_authenticated = True
+    mock_motionmount_config_flow.error_status = 0
+    assert await hass.config_entries.async_setup(mock_config_entry.entry_id)
+
+    assert hass.states.get("sensor.my_motionmount_error_status").state == "none"
+
+
+async def test_entity_rename(
+    hass: HomeAssistant,
+    device_registry: dr.DeviceRegistry,
+    entity_registry: er.EntityRegistry,
+    mock_config_entry: MockConfigEntry,
+    mock_motionmount_config_flow: MagicMock,
+) -> None:
+    """Tests the state attributes."""
+    mock_config_entry.add_to_hass(hass)
+
+    mock_motionmount_config_flow.name = ZEROCONF_NAME
+    mock_motionmount_config_flow.mac = MAC
+    mock_motionmount_config_flow.is_authenticated = True
```

Comment:
let's rename the mock to something shorter. Also, we can give that mock default so we don't have to set it every time